js/index.js

```diff
@@ -66,7 +68,9 @@ const add = document.querySelector('#add');
 Libraries.loadData();
 Libraries.displayData();
 add.addEventListener('click', () => {
-  Libraries.add(title.value, author.value);
-  title.value = '';
-  author.value = '';
+  if (title.value && author.value) {
+    Libraries.add(title.value, author.value);
+    title.value = '';
+    author.value = '';
```

Comment:
- [ ] OPTIONAL
The book _title_ and book _author_ input are both accepting **numbers** as inputs. Kindly endeavor to provide validation for the same.

Reply:
OK, thanks :heavy_check_mark: